docs/content/dagster-plus/account/managing-users/managing-user-roles-permissions.mdx

```diff
@@ -14,32 +14,56 @@ In this guide, we'll cover how RBAC works in Dagster+, how to assign roles to us
 
 ## Dagster+ user roles
 
-Dagster+ uses a hierarchical model for RBAC, meaning that the most permissive roles include permissions from the roles beneath them. The following user roles are currently supported, in order from the **most** permissive to the **least** permissive:
+Dagster+ employs a flexible approach to user roles and permissions. This system is built on two fundamental concepts:
 
-- Organization Admin
-- Admin
-- Editor
-- Launcher (Pro plans only)
-- Viewer
-
-For example, the **Admin** user role includes permissions specific to this role and all permissions in the **Editor**, **Launcher**, and **Viewer** user roles. Refer to the [User permissions reference](#user-permissions-reference) for the full list of user permissions in Dagster+.
+1. **Permission Scope** - Permissions are context-specific, falling into two main categories:
+   - Organization-wide settings (for example, "Create Teams")
+   - Deployment-specific actions (for example, "Launch and Cancel Backfills")
+2. **Role Types** - Dagster+ supports two types of roles:
+   - **Default Roles:** Hierarchical roles, based on sensible defaults that come out of the box with Dagster.
+   - **Custom Roles:** Allows you to define tailored roles with specific sets of permissions
 
+This dual approach provides both the simplicity of predefined roles and the flexibility to create custom roles that precisely match your organization's needs.
 ### User role enforcement
 
-All user roles are enforced both in Dagster+ and the [GraphQL API](/concepts/webserver/graphql).
+All roles are enforced both in Dagster+ and the [GraphQL API](/concepts/webserver/graphql).
 
 ### Teams
 
 Dagster+ Pro users can create teams of users and assign default permission sets. Refer to the [Managing teams in Dagster+](/dagster-plus/account/managing-users/managing-teams) guide for more info.
 
+--- 
+## Creating and editing custom roles
+
+In order to create or edit custom roles:
+1. Navigate to the **Organization Settings** page.
+2. Click the Roles tab
+
+Then to create a role:
+1. Click the **Create new role** button 
+2. Select a name and icon
+3. Provide a brief description
+4. Choose if deployment or organization type depending on where you want your role applied
+5. Select the permissions you want to apply to the role. Note - you can base your role off an existing role for ease of creation
+
+To edit a role:
+1. Click the edit button next to the role you want to edit
+2. Make your changes
+3. Save your changes
+4. You will see a confirmation dialog including the changes that have been made
 ---
 
-## Assigning user and team roles
+## Deleting custom roles
+Deleting custom roles is similar to editing. At the bottom of the edit dialog you'll see a button to "Delete role". Two things to note when deleting a role:
```

Comment:
I would go ahead and list the specific steps here, to make this section a bit easier to parse for people who haven't read through the sections above, and would change the formatting of the two notes a bit. Something like the following:

```
1. Navigate to the **Organization Settings** page.
2. Click the Roles tab.
3. Click the edit button next to the role you want to edit.
4. At the bottom of the edit dialog, click **Delete role**.

Note that if the role is currently assigned to any users, you will need to reassign them to a different role before deleting the role.

<Warning>Deleting a role is a permanent action and cannot be undone.</Warning>
```